UI/message.ts

```diff
@@ -58,6 +61,29 @@ function* match(regex: RegExp, content: string, type: ItemType): Generator<Conte
                     end: urlEndPosition,
                 };
             }
+        } else if (type == "nprofile") {
+            let bech32: string;
+            if (match[0].startsWith("nostr:")) {
+                bech32 = content.slice(urlStartPosition + 6, urlEndPosition + 1);
+            } else {
+                bech32 = content.slice(urlStartPosition, urlEndPosition + 1);
+            }
+            const decoded_nProfile = NostrProfile.decode(bech32);
+            if (decoded_nProfile instanceof Error) {
+                // ignore
+            } else {
+                const pubkey = PublicKey.FromBech32(decoded_nProfile.pubkey.hex);
+                if (pubkey instanceof Error) { //ignore
+                } else {
+                    yield {
+                        type: "npub",
```

Comment:
Not a blocker. Why not `nprofile`?

Reply:
my thought is, npub and nprofile is same ItemType when rendering
all pubkey + relays(optional)
the difference is the decode process, but the decode result is same type

Reply:
Well thought